packages/components/stories/vertical-tabs/vertical-tabs.stories.js

```diff
@@ -98,8 +66,6 @@ stories.add(
                 <div className='vertical-tabs-component'>
                     <VerticalTab
                         header_title='Header'
-                        action_bar={action_bar_items}
```

Comment:
@yauheni-deriv action_bar is used inside VerticalTab, please restore

Reply:
I don’t see any usage of this prop. it is not passed anywhere

Reply:
this prop is passed into <VerticalTab in stories, please leave it, by right stories should be also maintained, no need to remove a feature from this component😊

Reply:
action_bar prop has been restored